index.js

```diff
@@ -190,9 +192,13 @@ function choo () {
 // initial application state model
 // obj -> obj
 function appInit (opts) {
+  const initialLocation = (opts.hash === true)
+    ? hashMatch(document.location.hash)
+    : document.location.href
+
   const model = {
     namespace: 'app',
-    state: { location: document.location.href },
+    state: { location: initialLocation },
     subscriptions: [],
     reducers: {
       // handle href links
```

Comment:
Not sure the `.replace(/#.*/, '')` is necessary anymore with hash-match in use, though I'm not sure what purpose this served originally if it was meant to be used with browser history.